script.js

```diff
@@ -0,0 +1,148 @@
+// 1. Implement the getGreeting function.
+
+function getGreeting(firstName, lastName) {
+    return 'Hi, ' + firstName + ' ' + lastName + '. What is up?';
```

Comment:
This does not return the exact string mentioned in the exercise. Take a closer look.